artifacts/artifact.py

```diff
@@ -82,6 +82,11 @@ def AppendSource(self, type_indicator, attributes):
       raise errors.FormatError(
           u'Unsupported type indicator: {0}.'.format(type_indicator))
 
-    source_object = source_type_class(**attributes)
+    try:
+      source_object = source_type_class(**attributes)
```

Comment:
Please make consistent with the rest of the code base e.g.
https://github.com/ForensicArtifacts/artifacts/blob/2fb0c26b355556e7a9b0db8af4b13d0bc0b56a26/artifacts/reader.py#L114